src/MergePlugin.php

```diff
@@ -135,6 +144,21 @@ public static function getSubscribedEvents()
     }
 
     /**
+     * Handle an event callback for initialization.
+     *
+     * @param \Composer\EventDispatcher\Event $event
+     */
+    public function onInit(\Composer\EventDispatcher\Event $event)
```

Comment:
I'd suggest importing `Composer\EventDispatcher\Event` as `BaseEvent` to match how similar name collision issues are handled in the Composer codebase. I think it would be reasonable to change the `use Composer\Script\Event;` import to `use Composer\Script\Event as ScriptEvent;` as well.